ci/shared/scripts/setup-amd64-ubuntu-18.04-gcc-8-gcc-8-cpp.sh

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env sh
```

Comment:
We don't need to test Ubuntu 18.04 anymore and gcc-8 support is being removed.

Reply:
Image has been removed